presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaPageSource.java

```diff
@@ -249,10 +251,21 @@ private void addColumnsRecords(QualifiedTablePrefix prefix)
         for (Map.Entry<SchemaTableName, List<ColumnMetadata>> entry : listTableColumns(session, metadata, accessControl, prefix).entrySet()) {
             SchemaTableName tableName = entry.getKey();
             int ordinalPosition = 1;
+            String columnDisplayName = "";
             for (ColumnMetadata column : entry.getValue()) {
                 if (column.isHidden()) {
                     continue;
                 }
+                if (column.getType() instanceof TimestampType && ((TimestampType) column.getType()).getPrecision() == TimestampType.DEFAULT_PRECISION) {
```

Comment:
We should introduce a configuration option / session property to toggle this behavior. It will be temporary until clients get a chance to update their logic to handle variable-precision types.

Reply:
Working on this change now.

Reply:
Made the changes to allow session level parameter to control this option.